docs/Events/DeployEvent/Readme.md

```diff
@@ -0,0 +1,52 @@
+# dta event send deploy
+
+The sub-command `dta event send deploy` sends a deployment event to Dynatrace that updates the [release inventory](https://www.dynatrace.com/support/help/platform-modules/cloud-automation/release-monitoring/monitor-releases-with-dynatrace).
+
+### Getting started with deployment events
+
+1. Go to Dynatrace and find your application Process Group Instance (PGI) in the UI. Locate the combination of tags that uniquely identify your PGI. This information will be used in the entity selector.
+
+   Example image:
+
+   <img src="./assets/entity-selector.png"  width="375" height="200">
+
+1. Run the following command to trigger a new deployment event.
+
+```(bash)
+dta event deploy --entity-selector "type(PROCESS_GROUP_INSTANCE),tag(your-pgi-tag)" --name "app-deploy-v1-example" --version "v1.0.0
+```
+
+1. Navigate into Dynatrace and check the release inventory being update with the latest version.
+
+### Required Values
+
+The required values for this command are:
+
+| Command           | Environment variable  | Description                                                                             |
+| ----------------- | --------------------- | --------------------------------------------------------------------------------------- |
+| --entity-selector | EVENT_ENTITY_SELECTOR | Dynatrace entity selector expression : i.e.type(PROCESS_GROUP_INSTANCE),tag(easytravel) |
+| --name            | EVENT_DEPLOY_NAME     | Deployment event name. i.e. App-simplenode-Release-1.0.0                                |
+|                   |
+
+### Optional Values
+
+The optional values for this command are:
+
+| Command                   | Environment variable                 | Description                                                                                                                    |
+| ------------------------- | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------ |
+| --version                 | EVENT_DEPLOY_VERSION                 | Deployment version. i.e. 1.0.0. default: ""                                                                                    |
```

Comment:
How do you feel about aligning these vars and env vars with Dynatrace best practices?

https://www.dynatrace.com/support/help/platform-modules/cloud-automation/release-monitoring/version-detection-strategies

Reply:
I'm 50/50 between using the DT standard and using the current values. The current values are based on the properties available for an Dyntrace v2 event. The variable names in the best practices are different and are less than the current available options in the event. If we go for the DT best practices, should we remove the other variable options?, for example there is project and release project name available in the event API V2 but not in the DT release best practices.